policyengine_us/parameters/gov/states/nj/tax/income/exclusions/retirement/pension/agi_subtractions.yaml

```diff
@@ -1,9 +1,28 @@
 description: Deductions from AGI to calculate qualifying income for pension retirement exclusion.
 values:
   2021-01-01:
-    - tax_exempt_interest_income
-    - tax_exempt_pension_income
-    - taxable_social_security
+    - tax_exempt_interest_income # Line 16b
+    - social_security_retirement # Line 20a
```

Comment:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/18d1f826-6766-4b8c-9728-53e9450a2ff6)
Aren't only line 28 subtractions? e.g. 16b and 20a should be just add on to income

Reply:
<img width="379" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/104268340/e8fa4313-dd74-4bf1-8a6d-de3bf087b2f6">
<img width="383" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/104268340/4ab4dec0-b40c-4024-8704-47f046cd0144">
 the income limit we used is line 27, which do not include 16b and 20b

Reply:
social_security_retirement # Line 20a This is the content of Line 20b , but it was recorded on line 20a
<img width="772" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/104268340/63ff2437-5c85-4422-a2a0-c953b0149742">

Reply:
Yes but I think the agi_subtractions.yaml file should only represent the line 28, there are non-taxable income types (income that is completely excluded) and subtractions which are included in the federal computation and then taken out in the state one